### ISSUE TITLE
Hierarchy update

### ISSUE THREAD

**Issue by fatousylla** [medic/config-msf#104]
Adding a new step  in the hierarchy "verification step by a doctor" as presented by @mmureithi

**Comment by kennsippell** [comment]
Agreed in standup today that doctor verification steps won't be built as part of the pilot.
